src/DurableTask.Netherite/OrchestrationService/PartitionManagementOptions.cs

```diff
@@ -18,13 +18,20 @@ public enum PartitionManagementOptions
         ClientOnly,
 
         /// <summary>
-        /// Use the event processor host implementation provided by the EventHubs client library.
+        /// Use the event processor host implementation provided by the EventHubs client library. This dynamically
+        /// balances the partitions across all connected hosts.
         /// </summary>
         EventProcessorHost,
 
         /// <summary>
         /// Follow a predefined partition management script. This is meant to be used for testing and benchmarking scenarios.
+        /// This was an internal feature and is no longer supported.
         /// </summary>
         Scripted,
+
+        /// <summary>
+        /// Test the recovery without modifying storage. Useful for diagnosing recovery problems. Not functional as an orchestration service!
+        /// </summary>
+        RecoveryTester,
```

Comment:
nit: I wonder if adding `[Obsolete]` here would also prevent usage in prod.

```suggestion
        /// <summary>
        /// Test the recovery without modifying storage. Useful for diagnosing recovery problems. Not functional as an orchestration service! Marked as 'Obsolete' to prevent usage in production.
        /// </summary>
        [Obsolote]
        RecoveryTester,
```

Reply:
I prefer not to since it is not actually obsolete. 

I don't think I have ever seen anybody run into trouble because they are using undocumented configuration settings. I think they know better. There are a ton of such settings, and they are not currently marked obsolete.

Reply:
Fair enough, resolving.